archivist/notebooks/Find Asset and Event Creation.ipynb

```diff
@@ -338,7 +338,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.15"
+   "version": "3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \n[Clang 6.0 (clang-600.0.57)]"
```

Comment:
Is this too much information? Does the build data add anything beyond the version string

Reply:
I think this is auto-generated when you run the notebook. I guess on Mac taht is what you get